index.js

```diff
@@ -36,7 +40,8 @@ function Router (opts) {
   this.ignoreTrailingSlash = opts.ignoreTrailingSlash || false
   this.maxParamLength = opts.maxParamLength || 100
   this.allowUnsafeRegex = opts.allowUnsafeRegex || false
-  this.tree = new Node()
+  this.versioning = opts.versioning || versioningStrategies.ACCEPT_VERSION
```

Comment:
I don't think we need an object for the `versioningStrategies` since there is just one, you can pass directly `acceptVersion`.

Reply:
done